### DISCUSSION TITLE
Hotkey for Trash left/right

### DISCUSSION THREAD

**Discussion by faceleg** [theophanemayaud/video-simili-duplicate-cleaner#73] [Ideas]
I've got 10's of thousands of duplicate videos, this app is great.

Suggest optional hotkeys for trashleft/right, e.g. left/right arrow keys.

**Comment by theophanemayaud** [comment]
Thanks :) 
I can add this, I'm thinking of a toggle to enable such a behavior. And then use left/right arrows for quick deletion of left/right video, and maybe tab to skip to next pair. Sounds ok ?

**Comment by theophanemayaud** [comment]
Hey @faceleg, I finally got around to implementing this hotkey with #83 !
The feature which is new in the v1.8.0 update should be available within a few days when Microsoft and Apple approve it. I implemented left/right arrows for deleting left/right video, but also top/bottom arrows for going to previous/next video
Hope this helps, and thanks again for the great suggestion
